express-post-json/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+
+var nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+
+  gradeArray.push(grades);
+
+  res.json(gradeArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades/', (req, res) => {
+  grades[nextId] = {
+    id: nextId,
+    name: req.body.name,
+    course: req.body.course,
+    score: req.body.score
+  };
+  nextId++;
+  res.send(grades);
```

Comment:
Remember to use `.json` over `.send` when sending an object or array. Also you need to set the status code to `201` which means "Created". So it would look something like:

```JAVASCRIPT
res.status(201).json(grades);
```